bigquery/query_property_places.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: default
+title: Property of Places
+nav_order: 2
+parent: DC to BQ Sample Queries
+grand_parent: BigQuery
+---
+
+# Query Category: Properties of Places (and other entities)
```

Comment:
nit: Should these be Variables / Statistics rather than properties ? (since that's what the queries all deal with)

Reply:
Made an update. See if you prefer this?